src/js/rx/containers/Active.jsx

```diff
@@ -72,7 +72,9 @@ class Active extends React.Component {
       content = (
         <p className="rx-tab-explainer rx-loading-error">
           We couldn't retrieve your prescriptions.
-          Please refresh this page or try again later.
+          Please refresh this page or try again later. <br/>
```

Comment:
Without running this locally and seeing what styles these classes are actually adding, it seems like what you're trying to create with the `<br/>` is essentially 2 separate paragraphs. Why not wrap each paragraph in its own `<p>` instead? It's slightly more semantic this way and using HTML tags they way they should be used. This means you'll have something like:
```
<div>
  <p className="rx-tab-explainer rx-loading-error">
    We couldn't retrieve your prescriptions. Please refresh this page or try again later.
  </p>
  <p className="rx-tab-explainer rx-loading-error">
    If this problem persists, please call the Vets.gov Help Desk at ....
  </p>
</div>
```
The wrapping `div` will be necessary because React doesn't like sibling tags without being wrapped. 

If this isn't possible or becomes tricky because of how things are styled no worries! Just a suggestion. Otherwise looks good to me!